### ISSUE TITLE
Cannot load type definitions

### ISSUE THREAD

**Issue by mskutle** [Nosto/nosto-react#215]
VSCode (and presumably the TypeScript compiler) can't find the type definitions, so I can't really use this package right now.

![image](https://github.com/user-attachments/assets/bc9fc8c5-dc70-4fa9-981c-3c934b149d79)

The issue seems to be related to the "exports" field in package.json overriding the "types" field.
[https://stackoverflow.com/questions/76211877/the-xxxx-library-may-need-to-update-its-package-json-or-typings-ts](https://stackoverflow.com/questions/76211877/the-xxxx-library-may-need-to-update-its-package-json-or-typings-ts)

I cloned the repo and updated the "exports" field, which solved the problem for me locally.

```typescript
"exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "import": "./dist/index.es.js",
      "require": "./dist/index.umd.js"
    }
  },
``` 

I tried opening a pull request, but seems like I don't have access to do that.

**Comment by timowestnosto** [comment]
hi @mskutle 
Apologies for the delay. I just created a PR for the fix here https://github.com/Nosto/nosto-react/pull/229

**Comment by timowestnosto** [comment]
Fix is available now in version 2.3.0

**Comment by mskutle** [comment]
Thanks! Works fine now 👍
